pkg/util/log/clog.go

```diff
@@ -930,6 +933,18 @@ func (sb *syncBuffer) rotateFile(now time.Time) error {
 		return err
 	}
 
+	// Redirect stderr to the current INFO log file in order to capture panic
+	// stack traces that are written by the Go runtime to stderr. Note that if
+	// --logtostderr is true we'll never enter this code path and panic stack
+	// traces will go to the original stderr as you would expect.
+	if sb.sev == Severity_INFO &&
+		logging.stderrThreshold > Severity_INFO &&
+		!logging.noStderrRedirect {
+		if err := syscall.Dup2(int(sb.file.Fd()), syscall.Stderr); err != nil {
```

Comment:
Note that there's a race condition in here: any concurrent print to stderr may cause the output to be straddled across the old file and the new. Not that we have many of those, but perhaps a comment here would be warranted. Also it breaks the guarantee that all our log messages have the same format (start with a timestamp, etc.)